internal/runbits/requirements/requirements.go

```diff
@@ -261,18 +269,24 @@ func (r *RequirementOperation) ExecuteRequirementOperation(requirementName, requ
 		}
 	}
 
-	// Print the result
 	if !hasParentCommit {
-		out.Print(locale.Tr("install_initial_success", pj.Source().Path()))
+		out.Notice(locale.Tr("install_initial_success", pj.Source().Path()))
 	}
 
-	if requirementVersion != "" {
-		out.Print(locale.Tr(fmt.Sprintf("%s_version_%s", ns.Type(), operation), requirementName, requirementVersion))
-	} else {
-		out.Print(locale.Tr(fmt.Sprintf("%s_%s", ns.Type(), operation), requirementName))
+	// Print the result
+	message := locale.Tr(fmt.Sprintf("%s_version_%s", ns.Type(), operation), requirementName, requirementVersion)
+	if requirementVersion == "" {
+		message = locale.Tr(fmt.Sprintf("%s_%s", ns.Type(), operation), requirementName)
 	}
-
-	out.Print(locale.T("operation_success_local"))
+	out.Print(&requirementsOutput{
```

Comment:
All `output.Print()` calls going forward should print an object that has `MarshalOutput()` and `MarshalStructured()` methods. That way, the correct thing is printed to the terminal depending on the output mode.